### DISCUSSION TITLE
Avatar cannot display image from google profile src

### DISCUSSION THREAD

**Discussion by TomZhong-au** [chakra-ui/chakra-ui#7396] [Q&A]
I use google OAuth2 to get the user profile and display it with chakra Avatar. However, this cannot be displayed due to configuration from google's side.
There are discussions and solutions on stackoverflow: 
https://stackoverflow.com/questions/40570117/http403-forbidden-error-when-trying-to-load-img-src-with-google-profile-pic/61042200#61042200

It requires adding an attribute to the img tag.
```
<img src="your-google-link-here" referrerpolicy="no-referrer"/>
```
Is there a way to add this to the underpin image tag of the chakra Avatar component? Or is there any work around. Thanks.

**Comment by nbouvrette** [comment]
@TomZhong-au I was able to fix it directly in Charkra by using this property on the `Avatar` component like this: `referrerPolicy={'no-referrer'}`
